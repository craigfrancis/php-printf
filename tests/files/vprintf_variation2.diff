     *** Testing vprintf() : with unexpected values for args argument ***
     
     -- Iteration 1 --
004- vprintf(): Argument #2 ($values) must be of type array, int given
004+ user_vprintf(): Argument #2 ($values) must be of type array, int given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 2 --
007- vprintf(): Argument #2 ($values) must be of type array, int given
007+ user_vprintf(): Argument #2 ($values) must be of type array, int given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 3 --
010- vprintf(): Argument #2 ($values) must be of type array, int given
010+ user_vprintf(): Argument #2 ($values) must be of type array, int given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 4 --
013- vprintf(): Argument #2 ($values) must be of type array, int given
013+ user_vprintf(): Argument #2 ($values) must be of type array, int given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 5 --
016- vprintf(): Argument #2 ($values) must be of type array, float given
016+ user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 6 --
019- vprintf(): Argument #2 ($values) must be of type array, float given
019+ user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 7 --
022- vprintf(): Argument #2 ($values) must be of type array, float given
022+ user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 8 --
025- vprintf(): Argument #2 ($values) must be of type array, float given
025+ user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 9 --
028- vprintf(): Argument #2 ($values) must be of type array, float given
028+ user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 10 --
031- vprintf(): Argument #2 ($values) must be of type array, null given
031+ user_vprintf(): Argument #2 ($values) must be of type array, null given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 11 --
034- vprintf(): Argument #2 ($values) must be of type array, null given
034+ user_vprintf(): Argument #2 ($values) must be of type array, null given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 12 --
037- vprintf(): Argument #2 ($values) must be of type array, bool given
037+ user_vprintf(): Argument #2 ($values) must be of type array, bool given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 13 --
040- vprintf(): Argument #2 ($values) must be of type array, bool given
040+ user_vprintf(): Argument #2 ($values) must be of type array, bool given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 14 --
043- vprintf(): Argument #2 ($values) must be of type array, bool given
043+ user_vprintf(): Argument #2 ($values) must be of type array, bool given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 15 --
046- vprintf(): Argument #2 ($values) must be of type array, bool given
046+ user_vprintf(): Argument #2 ($values) must be of type array, bool given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 16 --
049- vprintf(): Argument #2 ($values) must be of type array, string given
049+ user_vprintf(): Argument #2 ($values) must be of type array, string given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 17 --
052- vprintf(): Argument #2 ($values) must be of type array, string given
052+ user_vprintf(): Argument #2 ($values) must be of type array, string given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 18 --
055- vprintf(): Argument #2 ($values) must be of type array, string given
055+ user_vprintf(): Argument #2 ($values) must be of type array, string given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 19 --
058- vprintf(): Argument #2 ($values) must be of type array, string given
058+ user_vprintf(): Argument #2 ($values) must be of type array, string given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 20 --
061- vprintf(): Argument #2 ($values) must be of type array, sample given
061+ user_vprintf(): Argument #2 ($values) must be of type array, sample given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 21 --
064- vprintf(): Argument #2 ($values) must be of type array, null given
064+ user_vprintf(): Argument #2 ($values) must be of type array, null given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 22 --
067- vprintf(): Argument #2 ($values) must be of type array, null given
067+ user_vprintf(): Argument #2 ($values) must be of type array, null given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
     
     -- Iteration 23 --
070- vprintf(): Argument #2 ($values) must be of type array, resource given
070+ user_vprintf(): Argument #2 ($values) must be of type array, resource given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
