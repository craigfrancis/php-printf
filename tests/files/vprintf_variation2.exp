*** Testing vprintf() : with unexpected values for args argument ***

-- Iteration 1 --
vprintf(): Argument #2 ($values) must be of type array, int given

-- Iteration 2 --
vprintf(): Argument #2 ($values) must be of type array, int given

-- Iteration 3 --
vprintf(): Argument #2 ($values) must be of type array, int given

-- Iteration 4 --
vprintf(): Argument #2 ($values) must be of type array, int given

-- Iteration 5 --
vprintf(): Argument #2 ($values) must be of type array, float given

-- Iteration 6 --
vprintf(): Argument #2 ($values) must be of type array, float given

-- Iteration 7 --
vprintf(): Argument #2 ($values) must be of type array, float given

-- Iteration 8 --
vprintf(): Argument #2 ($values) must be of type array, float given

-- Iteration 9 --
vprintf(): Argument #2 ($values) must be of type array, float given

-- Iteration 10 --
vprintf(): Argument #2 ($values) must be of type array, null given

-- Iteration 11 --
vprintf(): Argument #2 ($values) must be of type array, null given

-- Iteration 12 --
vprintf(): Argument #2 ($values) must be of type array, bool given

-- Iteration 13 --
vprintf(): Argument #2 ($values) must be of type array, bool given

-- Iteration 14 --
vprintf(): Argument #2 ($values) must be of type array, bool given

-- Iteration 15 --
vprintf(): Argument #2 ($values) must be of type array, bool given

-- Iteration 16 --
vprintf(): Argument #2 ($values) must be of type array, string given

-- Iteration 17 --
vprintf(): Argument #2 ($values) must be of type array, string given

-- Iteration 18 --
vprintf(): Argument #2 ($values) must be of type array, string given

-- Iteration 19 --
vprintf(): Argument #2 ($values) must be of type array, string given

-- Iteration 20 --
vprintf(): Argument #2 ($values) must be of type array, sample given

-- Iteration 21 --
vprintf(): Argument #2 ($values) must be of type array, null given

-- Iteration 22 --
vprintf(): Argument #2 ($values) must be of type array, null given

-- Iteration 23 --
vprintf(): Argument #2 ($values) must be of type array, resource given