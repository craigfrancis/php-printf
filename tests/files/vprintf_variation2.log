
---- EXPECTED OUTPUT
*** Testing vprintf() : with unexpected values for args argument ***

-- Iteration 1 --
vprintf(): Argument #2 ($values) must be of type array, int given

-- Iteration 2 --
vprintf(): Argument #2 ($values) must be of type array, int given

-- Iteration 3 --
vprintf(): Argument #2 ($values) must be of type array, int given

-- Iteration 4 --
vprintf(): Argument #2 ($values) must be of type array, int given

-- Iteration 5 --
vprintf(): Argument #2 ($values) must be of type array, float given

-- Iteration 6 --
vprintf(): Argument #2 ($values) must be of type array, float given

-- Iteration 7 --
vprintf(): Argument #2 ($values) must be of type array, float given

-- Iteration 8 --
vprintf(): Argument #2 ($values) must be of type array, float given

-- Iteration 9 --
vprintf(): Argument #2 ($values) must be of type array, float given

-- Iteration 10 --
vprintf(): Argument #2 ($values) must be of type array, null given

-- Iteration 11 --
vprintf(): Argument #2 ($values) must be of type array, null given

-- Iteration 12 --
vprintf(): Argument #2 ($values) must be of type array, bool given

-- Iteration 13 --
vprintf(): Argument #2 ($values) must be of type array, bool given

-- Iteration 14 --
vprintf(): Argument #2 ($values) must be of type array, bool given

-- Iteration 15 --
vprintf(): Argument #2 ($values) must be of type array, bool given

-- Iteration 16 --
vprintf(): Argument #2 ($values) must be of type array, string given

-- Iteration 17 --
vprintf(): Argument #2 ($values) must be of type array, string given

-- Iteration 18 --
vprintf(): Argument #2 ($values) must be of type array, string given

-- Iteration 19 --
vprintf(): Argument #2 ($values) must be of type array, string given

-- Iteration 20 --
vprintf(): Argument #2 ($values) must be of type array, sample given

-- Iteration 21 --
vprintf(): Argument #2 ($values) must be of type array, null given

-- Iteration 22 --
vprintf(): Argument #2 ($values) must be of type array, null given

-- Iteration 23 --
vprintf(): Argument #2 ($values) must be of type array, resource given
---- ACTUAL OUTPUT
*** Testing vprintf() : with unexpected values for args argument ***

-- Iteration 1 --
user_vprintf(): Argument #2 ($values) must be of type array, int given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 2 --
user_vprintf(): Argument #2 ($values) must be of type array, int given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 3 --
user_vprintf(): Argument #2 ($values) must be of type array, int given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 4 --
user_vprintf(): Argument #2 ($values) must be of type array, int given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 5 --
user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 6 --
user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 7 --
user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 8 --
user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 9 --
user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 10 --
user_vprintf(): Argument #2 ($values) must be of type array, null given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 11 --
user_vprintf(): Argument #2 ($values) must be of type array, null given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 12 --
user_vprintf(): Argument #2 ($values) must be of type array, bool given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 13 --
user_vprintf(): Argument #2 ($values) must be of type array, bool given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 14 --
user_vprintf(): Argument #2 ($values) must be of type array, bool given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 15 --
user_vprintf(): Argument #2 ($values) must be of type array, bool given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 16 --
user_vprintf(): Argument #2 ($values) must be of type array, string given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 17 --
user_vprintf(): Argument #2 ($values) must be of type array, string given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 18 --
user_vprintf(): Argument #2 ($values) must be of type array, string given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 19 --
user_vprintf(): Argument #2 ($values) must be of type array, string given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 20 --
user_vprintf(): Argument #2 ($values) must be of type array, sample given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 21 --
user_vprintf(): Argument #2 ($values) must be of type array, null given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 22 --
user_vprintf(): Argument #2 ($values) must be of type array, null given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 23 --
user_vprintf(): Argument #2 ($values) must be of type array, resource given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80
---- FAILED
