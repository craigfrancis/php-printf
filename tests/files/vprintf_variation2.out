*** Testing vprintf() : with unexpected values for args argument ***

-- Iteration 1 --
user_vprintf(): Argument #2 ($values) must be of type array, int given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 2 --
user_vprintf(): Argument #2 ($values) must be of type array, int given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 3 --
user_vprintf(): Argument #2 ($values) must be of type array, int given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 4 --
user_vprintf(): Argument #2 ($values) must be of type array, int given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 5 --
user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 6 --
user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 7 --
user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 8 --
user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 9 --
user_vprintf(): Argument #2 ($values) must be of type array, float given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 10 --
user_vprintf(): Argument #2 ($values) must be of type array, null given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 11 --
user_vprintf(): Argument #2 ($values) must be of type array, null given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 12 --
user_vprintf(): Argument #2 ($values) must be of type array, bool given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 13 --
user_vprintf(): Argument #2 ($values) must be of type array, bool given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 14 --
user_vprintf(): Argument #2 ($values) must be of type array, bool given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 15 --
user_vprintf(): Argument #2 ($values) must be of type array, bool given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 16 --
user_vprintf(): Argument #2 ($values) must be of type array, string given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 17 --
user_vprintf(): Argument #2 ($values) must be of type array, string given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 18 --
user_vprintf(): Argument #2 ($values) must be of type array, string given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 19 --
user_vprintf(): Argument #2 ($values) must be of type array, string given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 20 --
user_vprintf(): Argument #2 ($values) must be of type array, sample given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 21 --
user_vprintf(): Argument #2 ($values) must be of type array, null given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 22 --
user_vprintf(): Argument #2 ($values) must be of type array, null given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80

-- Iteration 23 --
user_vprintf(): Argument #2 ($values) must be of type array, resource given, called in /Volumes/WebServer/Projects/php.fprint/tests/files/vprintf_variation2.php on line 80